exercises/anagram/src/test/java/AnagramTest.java

```diff
@@ -52,6 +67,20 @@ public void testIdenticalWordIsNotAnagram() {
 
     @Ignore
     @Test
+    public void testCapitalWordIsNotAnagram() {
+        Anagram detector = new Anagram("BANANA");
+        assertTrue(detector.match(Arrays.asList("Banana")).isEmpty());
+    }
+
+    @Ignore
+    @Test
+    public void testIdenticalWordRepeatedIsNotAnagram() {
```

Comment:
Thanks; I think in the case where there's not a compelling reason to deviate from x-common data then that should be our default :)